src/core/src/bootstrap/Constants.py

```diff
@@ -160,6 +160,7 @@ class AutoAssessmentStates(EnumBackport):
     MAX_ASSESSMENT_RETRY_COUNT = 5
     MAX_INSTALLATION_RETRY_COUNT = 3
     MAX_IMDS_CONNECTION_RETRY_COUNT = 5
```

Comment:
Make the file retry count 10 please. And have it edited in the extension src constants too.

Reply:
Done